datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/imports.py

```diff
@@ -74,13 +76,14 @@ def imports(ctx, check, autofix):
     if validation_fails:
         num_files = len(validation_fails)
         num_failures = sum(len(lines) for lines in validation_fails.values())
-        echo_failure(f'\nValidation failed: {num_failures} deprecated imports found in {num_files} files:\n')
+        header_message = f'\nValidation failed: {num_failures} deprecated imports found in {num_files} files:\n'
+        echo_failure(header_message)
         for f, lines in validation_fails.items():
             for line in lines:
                 linenum, linetext = line
-                echo_warning(f'{f}: line # {linenum}', indent='  ')
+                echo_warning(f'{f}: line # {linenum + 1}', indent='  ')
```

Comment:
Why `linenum + 1`?

Reply:
the enumerated line num starts at 0, file lines start at 1.